### ISSUE TITLE
Possible to style multi-line code blocks

### ISSUE THREAD

**Issue by zeroSteiner** [Ionaru/easy-markdown-editor#270]
Right now multi-line code blocks that use the triple backquote show up in the editor with their contents colored, but whitespace and spacing between lines is the default. It's particularly distracting when there is indentation in the code as the whitespace doesn't have the same background. Is it possible to style these types of blocks to be just a consistent square?

![Screenshot from 2020-11-25 19-23-03](https://user-images.githubusercontent.com/2058303/100294524-0787d480-2f55-11eb-89fe-2272adca2ba8.png)

I'm not sure if this is possible strictly through CSS or if maybe a CodeMirror overlay could do the trick? I was really hoping there'd be a styleable CSS class added to each line that's part of the multi-line block. So far the best I've come up with is forcing the `.cm-comment` class to inherit it's background setting which removes it altogether.

```css
.cm-comment {
  background: inherit !important;
}
```

Thanks a lot for your work on this and any guidance you may be able to provide!

**Comment by stephencmorton** [comment]
Agreed. This would be very useful.

**Comment by DanielNetoP** [comment]
@Ionaru is this possible with css only or is there any need for custom code?

**Comment by Ionaru** [comment]
It may be possible to connect the lines using `line-height`, but a square/block around the code would need custom code.

**Comment by DanielNetoP** [comment]
Yeah, probably with a custom class on code blocks that we can then fully style the entire of the code mirror line right?

**Comment by DanielNetoP** [comment]
With the following css we're able to make the style across the entire editor, however if if have any break lines inside the code block that won't change it's color (the class doesn't exist there)
```
<style>
  .CodeMirror .CodeMirror-code .cm-comment:only-child {
    display: block;
}
</style>
```
<img width="418" alt="image" src="https://user-images.githubusercontent.com/10155766/177516442-5eadbb09-37ef-48b4-8b4a-a7360f86ed61.png">
